src/A1_csv_to_json.py

```diff
@@ -502,23 +576,19 @@ def add_storage_components(storage_filename, input_directory):
         dictionary containing the storage parameters
     """
 
-    if not os.path.exists(os.path.join(input_directory, "%s.csv" % storage_filename)):
-        logging.error("The storage file %s.csv" % storage_filename + " is missing!")
+    if not os.path.exists(os.path.join(input_directory, f"{storage_filename}.csv")):
+        logging.error(f"The storage file {storage_filename}.csv is missing!")
     else:
+        # hardcoded parameterlist of common parameters in all columns
```

Comment:
>  # hardcoded parameterlist of common parameters in all columns

As this list is shortened, would there still be an error message if "crate" is not in the storage_xx.csv file, ie. for the input/output power?